### ISSUE TITLE
Wishlist extraction doesn't work

### ISSUE THREAD

**Issue by joonaspaakko** [joonaspaakko/audible-library-extractor#152]
This has been the case for a long time... I didn't realize it didn't have an issue, so here it is.

**Comment by Ninjalord-25** [comment]
Is this the issue with the extractor button greyed out and unresponsive even in the audible library?

**Comment by joonaspaakko** [comment]
This issue is strictly related to the wishlist extraction failing.

I'm not sure I'm aware of one where anything is grayed out, but If you're talking about the extension icon, I think that may be related to issue #151, which is about the inability to start any extraction because the Audible Library Extractor button in the library is missing (because something changed since Chromium v130 and the extension fails to start. 

> I mention a workaround there, but there are so many other issues with the extraction process thanks to Audible fiddling with the website, I probably wouldn't bother with that at the moment.

**Comment by Ninjalord-25** [comment]
Yeah. I found the workarounds in question but it sadly didn't work so I'll just wait it out. Thank ya though!

**Comment by Ninjalord-25** [comment]
When i downloaded the different browsers and tried using the extension on them, it still gave me the same grayed out button it did on chrome. I don't know if I downloaded the wrong one but I tried brave and vivaldi and neither of them worked

**Comment by joonaspaakko** [comment]
@Ninjalord-25, would you mind taking a look at [the current v.0.2.11 draft](https://github.com/joonaspaakko/audible-library-extractor/raw/refs/heads/master/audible-library-extractor-0.2.11.zip)? I believe all the major bugs have been addressed, and it should work in any new Chromium versions.

**Comment by Ninjalord-25** [comment]
sorry, just now was able to get onto my computer to check it out! but i got it installed and it seems to be working with that update!
